uber/templates/base.html

```diff
@@ -219,7 +159,39 @@
                             </ul>
                         </li>
                     </ul>
-                    <ul class="nav navbar-nav" id="main-menu"></ul>
+                    {% block mainmenu %}
+                    <ul class="nav navbar-nav" id="main-menu">
+                        {% block mainmenu_items %}
```

Comment:
I'm not totally sure this extra `block` is needed, but I don't feel strongly about it either way.